packages/gluestick-cli/src/completion.js

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env node
+
+/** for dev testing:
+  source ../bin/completion.sh && GS_COMP=$PWD
+*/
+const { join } = require('path');
+const { existsSync, readdirSync } = require('fs');
+const commanderGlobal = require('./cli');
+
+let commanderProject = null;
+let entriesJson = null;
+let entriesPaths = [];
+
+function commanderBases(commander) {
+  return commander.commands
+    .map(comm => comm._name)
+    .filter(name => name !== '*')
+    .sort();
+}
+
+function commanderArgs(commander, name) {
+  const command = commander.commands.find(comm => comm._name === name);
+  return command ? command._args : [];
+}
+
+function commanderOpts(commander, name) {
+  const command = commander.commands.find(comm => comm._name === name);
+  return command
+    ? command.options.reduce(
+        (opts, curr) => opts.concat([curr.short, curr.long].filter(v => !!v)),
+        [],
+      )
+    : [];
+}
+
+function loadCommanderProject(cwd) {
+  if (!commanderProject) {
+    const completionVersion = '1.14'; // TODO: update me
```

Comment:
What is this version? Should it be `gluestick-cli` or `gluestick`??

Reply:
This checks the project version, so `gluestick`.  To be honest I don't know how how the publishing process works - I guess I assumed that one release of the project updated all packages to the same version.  Do I have that wrong?

Reply:
I don't really know which version you want, but if you need the latest version from npm then you can do sth similar to: https://github.com/TrueCar/gluestick/blob/next/packages/gluestick-cli/src/new.js#L12-L17

but if you need installed version of glustick from project then you can find it here
https://github.com/TrueCar/gluestick/blob/next/packages/gluestick-cli/src/cli.js#L24-L35

Reply:
Oh, I misunderstood - yes I check the locally installed version below, and this value is compared against it.  It needs to be the version in which these changes are released.

The purpose is to avoid trying to import the local commander file when it contains `commander.parse`.  That could happen with and updated version of `gluestick-cli` trying to import in a project with an older local package dependency.